packages/web-app/src/hooks/useOgMetaTags.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+
+export const useAddOgMetaTags = (ogMetaTagsRecord: Record<string, string>) => {
+  useEffect(() => {
+    Object.keys(ogMetaTagsRecord).forEach((ogMetaTagPropertyName) => {
+      const ogMetaTag = document.querySelector(`meta[property='og:${ogMetaTagPropertyName}']`)
+      const ogMetaTagContent = ogMetaTagsRecord[ogMetaTagPropertyName] as string
+
+      if (ogMetaTag) {
+        ogMetaTag.setAttribute('content', ogMetaTagContent)
+      } else {
+        const newOgMetaTag = document.createElement('meta')
+        newOgMetaTag.setAttribute('property', `og:${ogMetaTagPropertyName}`)
+        newOgMetaTag.setAttribute('content', ogMetaTagContent)
+        document.head.appendChild(newOgMetaTag)
+      }
+    })
+  }, [ogMetaTagsRecord])
+}
```

Comment:
1. Let's add some default meta tags values
2. let's rename `ogMetaTagsRecord` -> `ogMetaTags`  to avoid including data type in the variable name 

```suggestion
import { useEffect } from 'react'

const defaultOgMetaTags = {
  url: window.location.href,
  image: `${window.location.origin}/og-official-store.png`,
}

export const useOgMetaTags = (ogMetaTags: Record<string, string>) => {
  useEffect(() => {
    Object.keys({ ...ogMetaTags, ...defaultOgMetaTags }).forEach((ogMetaTagPropertyName) => {
      const ogMetaTag = document.querySelector(`meta[property='og:${ogMetaTagPropertyName}']`)
      const ogMetaTagContent = ogMetaTags[ogMetaTagPropertyName] as string

      if (ogMetaTag) {
        ogMetaTag.setAttribute('content', ogMetaTagContent)
      } else {
        const newOgMetaTag = document.createElement('meta')
        newOgMetaTag.setAttribute('property', `og:${ogMetaTagPropertyName}`)
        newOgMetaTag.setAttribute('content', ogMetaTagContent)
        document.head.appendChild(newOgMetaTag)
      }
    })
  }, [ogMetaTags])
}

```

Reply:
I agree with URL but I think image shouldn't be included in default tags

Reply:
Displaying the default image in case there wasn't some specific passed - sounds correct to me, any reason to avoid this behavior?

Reply:
I thought you proposed changing every image to the default. Displaying the default image in case there wasn't some specific passed is a good idea. Changed